config/config.go

```diff
@@ -494,8 +494,8 @@ type SyncToHostCustomResource struct {
 	// Enabled defines if this option should be enabled.
 	Enabled bool `json:"enabled,omitempty"`
 
-	// Translate the patch according to the given patches.
-	Translate []TranslatePatch `json:"translate,omitempty"`
+	// Patches the resources according to the given patches.
+	Patches []TranslatePatch `json:"patches,omitempty"`
 }
 
 type TranslatePatch struct {
```

Comment:
Do we want to rename this to just "Patch"?

Reply:
No, it's an array and multiple patches can be used.

Reply:
I'm referring to the struct not the field name. There is also the struct ` EnableSwitchWithTranslate` which should probably be changed to `EnabledSwitchWithPatch`.

Reply:
"TranslatePatch" as struct is fine, as internally we perform translation operations (see the translate package in pkg).